app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -107,20 +116,39 @@ class HomeViewModel(
   }
 
   /**
-   * Returns a [HomeItemViewModel] corresponding to the promoted stories to be displayed for this learner
-   * (see [PromotedStoryListViewModel]), or null if this profile does not have any promoted stories.
-   * Promoted stories are determined by any recent stories started by this profile.
+   * Returns a [HomeItemViewModel] corresponding to the promoted stories(Recommended, Recently-played and
+   * Last-played stories)[PromotedStoryListViewModel] and Upcoming topics [ComingSoonTopicListViewModel]
```

Comment:
Should this be "Last-played stories) [PromotedStoryListViewModel] **or** Upcoming topics [ComingSoonTopicListViewModel]"? Because we will show either the promoted stories or the coming soon topics, but never both of these at the same time.